ingest-app/docs/api.md

```diff
@@ -1192,6 +1197,64 @@ Ingest permissions for granule subscriptions are granted through the provider vi
 For lack of a better ACL, ingest permissions for collection subscription are granted through the SYSTEM OBJECT TAG_GROUP ACL update permission.
 
 %GENERIC-DOCS%
+
+## <a name="publish-drafts"></a> Publish Drafts 
+### <a name="publish-non-variable-draft"></a> Publish Non Variable Draft 
+#### <a name="publish-non-variable-draft-endpoint"></a> /publish/&lt;non-variable-draft-concept-id&gt;/&lt;native-id&gt;
+
+Non variable drafts can be published, i.e. ingested into CMR as non variable concept, through the non variable draft publishing endpoint.
```

Comment:
Change the text to read:
All drafts can be published, i.e. ingested into the CMR as a new concept through the publishing endpoints. This specific endpoint publishes all draft records except for variables drafts. Variable drafts can be published through the next described endpoint.

Reply:
done.